clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -321,11 +321,11 @@ CompletableFuture<ClientResponse> future() {
             return handler.future;
         }
 
-        FutureCompletionHandler handler() {
+        public FutureCompletionHandler handler() {
```

Comment:
I had to loose the visibility of some classes. I am not sure we are fine with that. WDYT?